backend/src/main/java/com/wpz/rbs/model/reservation/CreateReservation.java

```diff
@@ -0,0 +1,27 @@
+package com.wpz.rbs.model.reservation;
+
+import jakarta.validation.constraints.NotEmpty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class CreateReservation {
+    @NotEmpty
+    private String name;
+    @NotEmpty
+    private String email;
+    @NotEmpty
+    private String first_name;
+    @NotEmpty
+    private String last_name;
+    @NotEmpty
+    private String start_time;
+    @NotEmpty
+    private String end_time;
+    private int room_id;
```

Comment:
can this be empty?

Reply:
Yes, bcs int isn't nullable and throws error when not provided, but I will add @NotEmpty for clearness